CHANGELOG.md

```diff
@@ -2,10 +2,16 @@
 
 ## dev
 
+
```

Comment:
Do we normally have two lines below `##dev`?

Reply:
im not sure lol. it happened when i was resolving a merge conflict and i just didnt go back and remove it. I have changed it now though lol